### ISSUE TITLE
flake: test/cmd/sdn.sh:27: executing 'oc get netnamespace default -o jsonpath="{.netid}"' expecting success and text '^0$'

### ISSUE THREAD

**Issue by abstractj** [openshift/origin#15457]
```
github.com/openshift/origin/test/cmd/sdn/netnamespaces.test/cmd/sdn.sh:27: executing 'oc get netnamespace default -o jsonpath="{.netid}"' expecting success and text '^0$' (from github.com_openshift_origin_test_cmd_sdn_netnamespaces)

Failing for the past 1 build (Since Failed#3328 )
Took 0.6 sec.
Error Message

the command returned the wrong error code; the output content test failed
Stacktrace

=== BEGIN TEST CASE ===
test/cmd/sdn.sh:27: executing 'oc get netnamespace default -o jsonpath="{.netid}"' expecting success and text '^0$'
FAILURE after 0.600s: test/cmd/sdn.sh:27: executing 'oc get netnamespace default -o jsonpath="{.netid}"' expecting success and text '^0$': the command returned the wrong error code; the output content test failed
There was no output from the command.
Standard error from the command:
Error from server (NotFound): netnamespaces.network.openshift.io "default" not found
=== END TEST CASE ===
```

Happened in Job: https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin/3328/
Seen here: https://github.com/openshift/origin/pull/15314

**Comment by enj** [comment]
@openshift/networking

**Comment by pweil-** [comment]
same as https://github.com/openshift/origin/issues/15337 and was thought to be fixed by #15353

**Comment by liggitt** [comment]
the linked build ran on Jul 20, 2017 9:24:17 AM

the fix merged at Jul 20, 2017 3:47 PM
